client/src/layouts/Dashboard.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+
+function Dashboard() {
+  return (
+    <div className="container">
```

Comment:
as you changed the class name change it in the app.css also

Reply:
okay, I will do it now. 👨🏻‍💻